pkg/services/ngalert/notifier/channels_config/available_channels.go

```diff
@@ -815,6 +815,15 @@ func GetAvailableNotifiers() []*NotifierPlugin {
 					PropertyName: "chatid",
 					Required:     true,
 				},
+				{
+					Label:          "Message Thread ID",
+					Element:        ElementTypeInput,
+					InputType:      InputTypeText,
+					Description:    "Integer Telegram Message Thread Identifier",
+					PropertyName:   "messagethreadid",
```

Comment:
This field name should match https://github.com/grafana/alerting/pull/143/files#diff-ed9e160428d5b291f759bad53d5b04e320378f97a0340630c18041357fa44fa2R23

Reply:
Done.